app/views/featured_attachments/_featured_attachment.html.erb

```diff
@@ -1,4 +1,19 @@
+<% actions = [] %>
+<% actions << capture do %>
+  <%= form_tag(
+    file_attachment_path(document, attachment.file_attachment_id),
+    method: :delete,
+    class: "app-inline-block",
+    data: {
+      "modal-action": "delete",
```

Comment:
Minor: this is only applicable when this is used in a modal context. We can lose it here. It looks like it goes in a subsequent commit anyway.